edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/endpoints/StoringAsyncEndpointExecutorTest.cs

```diff
@@ -36,7 +36,7 @@ public async Task InvokeTest()
             // Act - Send messages to invoke
             foreach (IMessage message in messages)
             {
-                await storingAsyncEndpointExecutor.Invoke(message);
+                await storingAsyncEndpointExecutor.Invoke(message, 0, 3600);
```

Comment:
In general, should we use random number for TTL and priority for existing tests.  Rather than hard-coded a fixed value.